### ISSUE TITLE
systemjs/webpack issue

### ISSUE THREAD

**Issue by zladuric** [maxisam/ngx-clipboard#81]
Hi,

I have a weird configuration issue with this, maybe you can shed some light on my problems with webpack/systemjs config.

Specifically I use it with [this](https://github.com/mgechev/angular-seed) seed: git clone git@github.com:mgechev/angular-seed.git and npm install. Then npm install ngx-clipboard. Then, I add this to additional packages in project.config.ts (which sets up system.js config):

```{ name: 'ngx-clipboard', path: 'node_modules/ngx-clipboard/dist/bundles/ngxClipboard.umd.js' }```

This gives me a nice little entry in system.js config:

```"ngx-clipboard": "node_modules/ngx-clipboard/dist/bundles/ngxClipboard.umd.js"````

Now, npm start complains, it says it cannot find `ngx-window-token/index`. So I look where it is, and add this as an additional package:

```{ name: 'ngx-window-token', path: 'node_modules/ngx-clipboard/node_modules/ngx-window-token/dist/bundles/ngxWindowToken.umd.min.js' }```

This goes into system.js config as:

```"ngx-window-token": "node_modules/ngx-clipboard/node_modules/ngx-window-token/dist/bundles/ngxWindowToken.umd.min.js"```

But now this goes boom in another way: `Cannot GET /node_modules/ngx-clipboard/node_modules/ngx-window-token/dist/bundles/ngxWindowToken.umd.min.js/index.`.

Basically it always appends /index to whatever I setup nxg-window-token to. (Well I don't even do that. ngx-clipboard does.)

The same happens if I try to install ngx-window-token directly and adjusting the path there.

What could be wrong there? Does anybody have any pointers, where could I even look?

**Comment by zladuric** [comment]
@maxisam for what it's worth, in version `8.0.4`, the ngxClipboard.umd.js bundle packed this:

    require('ngx-window-token')

And for `8.1.0`, it packed:

    require('ngx-window-token/index');

Do you know why would that be the case?

**Comment by maxisam** [comment]
Could you try v8.1.1?

**Comment by zladuric** [comment]
Yep, that fixed the problem for me, thanks.
